typescript-functions/main.ts

```diff
@@ -0,0 +1,45 @@
+function convertMinutesToSeconds(minutes: number): number {
+  return minutes * 60;
+}
+console.log(convertMinutesToSeconds(5));
+
+function greet(name: string): string {
+  return 'Hi' + ' ' + name;
+}
+console.log(greet('chubbybunny'));
+
+const getArea = (width: number, height: number): number => {
+  return width * height;
+};
+console.log(getArea(17, 42));
+
+const getFirstName = (person: {
+  firstName: string;
+  lastName: string;
+}): string => {
+  return person.firstName;
+};
+
+interface person {
+  firstName: string;
+  lastName: string;
+}
+const katelyn: person = {
+  firstName: 'katelyn',
+  lastName: 'ngo',
+};
+
+console.log(getFirstName);
```

Comment:
You are logging the function itself, but you are not _calling_ the function. I would like to see you call the function so that we log the return to the console (you did this right on line 14!)